client/src/Routes.js

```diff
@@ -18,6 +20,16 @@ const Routes = () => {
       <Route path="/login" component={Login} />
       <Route path="/applicant_create_profile" component={ApplicantProfile} />
       <Route path="/profile_creation_successful" component={SuccessMessage} />
+      <Route
+        exact
+        path="/public_applicant_profiles"
```

Comment:
I can see that we're using underscores here for consistency because lines 21-22 above also use them, but I just wanted to mention out of interest that websites tend to use dashes in URLs.

Examples:

- https://reactjs.org/docs/hooks-state.html
- https://react.semantic-ui.com/collections/grid/#variations-column-width
- https://expressjs.com/en/starter/basic-routing.html

Reply:
blame with that starts with me because that's what I wrote in the ticket acceptance criteria and we decided to roll with it :)

Reply:
Resolved